Library/Homebrew/test/support/fixtures/bottles/testball_bottle-0.1.mountain_lion.bottle.tar.xz

```diff
@@ -0,0 +1 @@
+testball_bottle-0.1.yosemite.bottle.tar.xz
```

Comment:
Do we need a fixture for every version? Would be good to remove at least the need for mountain lion/yosemite fixtures.